src/components/v5/common/Fields/Select/Select.tsx

```diff
@@ -93,43 +122,61 @@ const Select = <T extends any[]>({
           <Icon name="caret-down" appearance={{ size: 'extraTiny' }} />
         </span>
       </button>
-      <ul
-        className={`${styles.options} ${isOptionsOpen ? styles.show : ''}`}
-        tabIndex={-1}
-        role="listbox"
-        aria-activedescendant={list[selectedElement].id.toString()}
-        onKeyDown={(event) => handleListKeyDown(event)}
-      >
-        {filteredList?.map((option) => (
-          <li
-            key={option.id}
-            className={clsx(styles.li, {
-              'text-blue-400 font-medium': selectedElement === option.id,
-              'text-gray-900': selectedElement !== option.id,
-            })}
-            id={option.id.toString()}
-            role="option"
-            aria-selected={selectedElement === option.id}
-            tabIndex={0}
-            onKeyDown={handleKeyDown(option.id)}
-            onClick={() => {
-              handleChange(option.id);
-              setIsOptionsOpen(false);
-            }}
-          >
-            {option.linkTo ? (
-              <NavLink
-                className="flex items-center w-full text-inherit py-2"
-                to={option.linkTo}
+      {!isLoading && (
+        <ul
+          className={`${isListRelative && 'relative'} ${styles.options} ${
+            isOptionsOpen ? styles.show : ''
+          }`}
+          tabIndex={-1}
+          role="listbox"
+          aria-activedescendant={
+            selectedElement && list[selectedElement].id.toString()
+          }
+          onKeyDown={(event) => handleListKeyDown(event)}
+        >
+          {optionsList &&
+            optionsList.map((option) => (
```

Comment:
Please, destructure the object and do this always in the future

Reply:
destructured